indexer/types.go

```diff
@@ -60,11 +60,12 @@ type AbiJob struct {
 	Status                string
 	HistoricalCrawlStatus string
 	Progress              int
-	MoonwormTaskPickedup  bool
+	TaskPickedup          bool
 	Abi                   string
 	AbiType               string
 	CreatedAt             time.Time
 	UpdatedAt             time.Time
+	DeploymentBlockNumber *uint64
```

Comment:
Do we need here pointer (*) maybe better to use as everywhere exact number to avoid divergence with other variables?